.gitignore

```diff
@@ -17,4 +17,7 @@
 tools/specFiles
 tools/*-local.json
 tools/*-local-raw.json
-tools/build
\ No newline at end of file
+tools/build
+
+# RustRover
+.idea/
```

Comment:
Like `.vscode`, we probably want to keep some IDE configurations in the version control.

Have a look at: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

Reply:
I know, but @noandrea said that for now it's best to leave it like this until we decide if we want to acknowledge RustRover or not.